src/humanize/locale/zh_HK/LC_MESSAGES/humanize.po

```diff
@@ -0,0 +1,365 @@
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:65
+msgctxt "8 (male)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:66
+msgctxt "9 (male)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:70
+msgctxt "0 (female)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:71
+msgctxt "1 (female)"
+msgid "st"
+msgstr "第"
+
+#: src/humanize/number.py:72
+msgctxt "2 (female)"
+msgid "nd"
+msgstr "第"
+
+#: src/humanize/number.py:73
+msgctxt "3 (female)"
+msgid "rd"
+msgstr "第"
+
+#: src/humanize/number.py:74
+msgctxt "4 (female)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:75
+msgctxt "5 (female)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:76
+msgctxt "6 (female)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:77
+msgctxt "7 (female)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:78
+msgctxt "8 (female)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:79
+msgctxt "9 (female)"
+msgid "th"
+msgstr "第"
+
+#: src/humanize/number.py:140
+msgid "thousand"
+msgid_plural "thousand"
+msgstr[0] "千"
+msgstr[1] "千"
+
+#: src/humanize/number.py:141
+msgid "million"
+msgid_plural "million"
+msgstr[0] "百萬"
+msgstr[1] "百萬"
+
+#: src/humanize/number.py:142
+msgid "billion"
+msgid_plural "billion"
+msgstr[0] "十億"
+msgstr[1] "十億"
+
+#: src/humanize/number.py:143
+msgid "trillion"
+msgid_plural "trillion"
+msgstr[0] "兆"
+msgstr[1] "兆"
+
+#: src/humanize/number.py:144
+msgid "quadrillion"
+msgid_plural "quadrillion"
+msgstr[0] "萬億"
+msgstr[1] "萬億"
+
+#: src/humanize/number.py:145
+msgid "quintillion"
+msgid_plural "quintillion"
+msgstr[0] "百京"
+msgstr[1] "百京"
+
+#: src/humanize/number.py:146
+msgid "sextillion"
+msgid_plural "sextillion"
+msgstr[0] "十垓"
+msgstr[1] "十垓"
+
+#: src/humanize/number.py:147
+msgid "septillion"
+msgid_plural "septillion"
+msgstr[0] "秭"
+msgstr[1] "秭"
+
+#: src/humanize/number.py:148
+msgid "octillion"
+msgid_plural "octillion"
+msgstr[0] "千秭"
+msgstr[1] "千秭"
+
+#: src/humanize/number.py:149
+msgid "nonillion"
+msgid_plural "nonillion"
+msgstr[0] "百穰"
+msgstr[1] "百穰"
+
+#: src/humanize/number.py:150
+msgid "decillion"
+msgid_plural "decillion"
+msgstr[0] "十溝"
+msgstr[1] "十溝"
+
+#: src/humanize/number.py:151
+msgid "googol"
+msgid_plural "googol"
+msgstr[0] "古高爾"
+msgstr[1] "古高爾"
+
+#: src/humanize/number.py:246
+msgid "zero"
+msgstr "零"
+
+#: src/humanize/number.py:247
+msgid "one"
+msgstr "一"
+
+#: src/humanize/number.py:248
+msgid "two"
+msgstr "二"
+
+#: src/humanize/number.py:249
+msgid "three"
+msgstr "三"
+
+#: src/humanize/number.py:250
+msgid "four"
+msgstr "四"
+
+#: src/humanize/number.py:251
+msgid "five"
+msgstr "五"
+
+#: src/humanize/number.py:252
+msgid "six"
+msgstr "六"
+
+#: src/humanize/number.py:253
+msgid "seven"
+msgstr "七"
+
+#: src/humanize/number.py:254
+msgid "eight"
+msgstr "八"
+
+#: src/humanize/number.py:255
+msgid "nine"
+msgstr "九"
+
+#: src/humanize/time.py:133
+#, python-format
+msgid "%d microsecond"
+msgid_plural "%d microseconds"
+msgstr[0] "微秒"
+msgstr[1] "微秒"
```

Comment:
Looks like these are missing the `%d` where the number will go.

Reply:
Hi @hugovk! Thanks for the reminder, will amend now.